app/connectors/InvitationsCacheConnector.scala

```diff
@@ -14,21 +14,110 @@
  * limitations under the License.
  */
 
-package connectors.cache.microservice
+package connectors
 
 import com.google.inject.Inject
 import config.FrontendAppConfig
-import connectors.MicroserviceCacheConnector
-import play.api.libs.ws.WSClient
-import uk.gov.hmrc.crypto.ApplicationCrypto
+import models.Invitation
+import play.api.http.Status._
+import play.api.libs.json._
+import play.api.libs.ws.{WSClient, WSRequest}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpException}
+
+import scala.concurrent.{ExecutionContext, Future}
 
 class InvitationsCacheConnector @Inject()(
```

Comment:
Could we have a trait defining this as per other connectors? Makes test doubles easier to write.

Reply:
Done